src/utils/changeImages.ts

```diff
@@ -10,10 +10,19 @@ const changeImagesToPepe = (imgs: HTMLImageElement[]) => {
   for (const img of imagesToChange){
     const url = getRandomPepeImage();
     img!.setAttribute("currentSrc", url);
+    img!.setAttribute("srcset", url);
     img!.src = url;
   }
 };
 
+const changeVideosToPepe = (videos : HTMLVideoElement[]) => {
+  for (const video of videos){
+    const url = runtime.getURL("videos/pepe_listening_music.gif");
```

Comment:
nit: could make it configurable via env vars by moving it to `config.ts` 
```suggestion
const url = runtime.getURL(config.videoToPepeFilePath);
```

Then

```
// config.ts
videoToPepeFilename = process.env.VIDEO_TO_PEPE_FILENAME;
``

```
// .env
REACT_APP_VIDEO_TO_PEPE_FILENAME = "videos/pepe_listening_music.gif"
```

For now we'll assume there's a single image to replace videos for